docs/reference/installation.rst

```diff
@@ -1,21 +1,32 @@
 .. index::
     single: Installation
+    single: Configuration
 
 Installation
 ============
 
 Prerequisites
 -------------
 
-PHP 7 and Symfony >=3.4 or >= 4.2 are needed to make this bundle work, there are
-also some Sonata dependencies that need to be installed and configured beforehand:
+PHP 7.2 and Symfony 4.4 are needed to make this bundle work, there are
```

Comment:
IMO, if we don't use range operators or explain that these are the minimum requirements, the message is telling that only these specific versions are allowed.
```suggestion
PHP ^7.2 and Symfony ^4.4 are needed to make this bundle work, there are
```

Reply:
The version in the docs soon become outdated. Can't we just remove the versions? Composer will tell what's missing.

Reply:
>The version in the docs soon become outdated. Can't we just remove the versions? Composer will tell what's missing.

I understand the point and agree, but I think this may lead to a poor experience for newcomers if they are not confident executing a command before being sure if the requirements are met.
Maybe we could aim the users to execute `composer require ...` and optionally target to https://packagist.org/packages/sonata-project/user-bundle in case they want to know about the requirements, since in this case Packagist will act as a documentation for our constraints.